x-pack/plugins/infra/public/pages/metrics/hosts/components/hosts_table_columns.tsx

```diff
@@ -10,28 +10,29 @@ import React from 'react';
 import { i18n } from '@kbn/i18n';
 import { EuiText } from '@elastic/eui';
 import { scaleUpPercentage } from '../../../../components/infrastructure_node_metrics_tables/shared/hooks';
-import { SnapshotNodeMetric } from '../../../../../common/http_api/snapshot_api';
 import { NumberCell } from '../../../../components/infrastructure_node_metrics_tables/shared/components';
 
-interface HostNodeRow extends HostMetics {
+export interface HostNodeRow extends HostMetics {
   os?: string | null | undefined;
   servicesOnHost?: number | null | undefined;
+  name: string;
 }
 
 export interface HostMetics {
```

Comment:
I changed the structure a bit - so for the sorting functionality we need a structure like `field_name: value (string/number)` and I was thinking that it's more readable if we have the name mapped separately (instead of using the label from the path)